### DISCUSSION TITLE
What ASN.1 compiler are you using?

### DISCUSSION THREAD

**Discussion by bhack** [srsran/srsRAN_Project#18] [General Help] [answered]
From https://github.com/srsran/srsRAN_Project/pull/15#issuecomment-1455697224
> RRC library is auto-generated from the 3GPP specs ASN1 code.

@andrepuschmann What compiler are you using?

**Comment by andrepuschmann** [comment, marked as answer]
> @andrepuschmann What compiler are you using?

We have developed an own asn1 code-generator.
